src/NServiceBus.Core/Transports/Learning/LearningTransportMessagePump.cs

```diff
@@ -35,6 +35,13 @@ public Task Init(Func<MessageContext, Task> onMessage, Func<ErrorContext, Task<E
             committedTransactionDir = Path.Combine(messagePumpBasePath, CommittedDirName);
 
             purgeOnStartup = settings.PurgeOnStartup;
```

Comment:
Do we still need this local now that we don't care about this value outside of `Init`?

Reply:
no, not needed anymore